gossip/minimal/configpb/config.proto

```diff
@@ -77,8 +79,8 @@ message GossipConfig {
 message GoshawkConfig {
   // The source logs whose STHs will be checked.
   repeated LogConfig source_log = 1;
-  // The destination hub which will be scanned for minimal-gossip certificates.
-  HubConfig dest_hub = 2;
+  // The destination hubs which will be scanned for minimal-gossip certificates.
+  repeated HubConfig dest_hub = 2;
```

Comment:
It's wire compatible but generated code that hasn't been updated will only see the last config. I think we are the only people deploying this anyway at the moment so it's safe.

Reply:
Yeah, this is all clearly marked as experimental and hasn't been deployed yet (to my knowledge).